lib/common.py

```diff
@@ -877,6 +900,13 @@ def get_all_efi_bins(self):
 
         return binaries
 
+    def clone(self):
+        name = self.name() + '_clone_for_tests'
+        print("Clone VM")
+        uuid = self.host.xe('vm-clone', {'uuid': self.uuid, 'new-name-label': "\"%s\"" % name})
```

Comment:
Perhaps the `xe` method should escape strings to avoid having to do thinkg like this?